services-queues.md

```diff
@@ -171,6 +172,43 @@ In addition, you may specify the number of seconds to wait before polling for ne
 
 Note that the queue only "sleeps" if no jobs are on the queue. If more jobs are available, the queue will continue to work them without sleeping.
 
+<a name="supervisor-configuration"></a>
+## Supervisor configuration
+
+### Installing Supervisor
+
+Supervisor is a process monitor for the Linux operating system, and will automatically restart your `queue:work` process if it fails. To install Supervisor on Ubuntu, you may use the following command:
+
+    sudo apt-get install supervisor
+
+### Configuring Supervisor
+
+Supervisor configuration files are typically stored in the `/etc/supervisor/conf.d` directory. Within this directory, you may create any number of configuration files that instruct supervisor how your processes should be monitored. For example, let's create a `october-worker.conf` file that starts and monitors a `queue:work` process:
+
+    [program:october-worker]
+    process_name=%(program_name)s_%(process_num)02d
+    command=php /path/to/october/artisan queue:work --sleep=3 --tries=3
+    autostart=true
+    autorestart=true
+    user=october
```

Comment:
Is this supposed to be the linux user that's serving October?

Reply:
`user=` just needs to be a user that has permission to run the `queue:work` command. Do you want that note added?

Reply:
Yes please.

Reply:
Added.